etc/travis/sylius18/bundles.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
These configs should not be in `etc/travis` directory as we don't use travis anymore on this repository